documents-webapp/src/main/webapp/vue-app/documents/components/body/actions/DownloadMenuAction.vue

```diff
@@ -29,8 +29,10 @@ export default {
   },
   methods: {
     download() {
+      console.log(this.file.id);
```

Comment:
could you remove the console.log statements ?

Reply:
done